master/internal/task/allocation.go

```diff
@@ -305,7 +305,8 @@ func (a *allocation) SetProxyAddress(_ context.Context, address string) error {
 	defer a.mu.Unlock()
 
 	if len(a.req.ProxyPorts) == 0 {
-		return ErrBehaviorUnsupported{Behavior: "proxying"}
+		a.syslog.Debug("No ports to proxy. Skipping proxy registration.")
```

Comment:
Is there any context here?   This kind of message is difficult to match up with any actual behavior unless the task/port# or something specific to the request is displayed.

Otherwise better to say nothing. IMO.

Reply:
The meaning of the message is that this was not specified in the config.

```
  proxy_ports:
    - proxy_port: 7860
      proxy_tcp: true
      unauthenticated: true
```

Yes, I could not display any message at all, but if you do have the proxy in your config and you're not getting your proxy registered, at least you will see the message in the log to give you an idea of what happened.